packages/vector_graphics/lib/src/vector_graphics.dart

```diff
@@ -376,13 +371,17 @@ class _VectorGraphicWidgetState extends State<VectorGraphic> {
   }
 
   void _handleError(Object error, StackTrace? stackTrace) {
+    if (!mounted) {
```

Comment:
Since this function is invoked _after_ the future has been initiated, the widget could be unmounted.